pywwise/ak/wwise/core/transport.py

```diff
@@ -1,54 +1,154 @@
 from simplevent import RefEvent as _RefEvent
 from waapi import WaapiClient as _WaapiClient
+from pywwise.enums import EObjectType, ETransportExecuteActions, ETransportState
+from pywwise.statics import EnumStatics
+from pywwise.structs import TransportObjectInfo
+from pywwise.types import GameObjectID, GUID, Name, ProjectPath
 
 
 class Transport:
-	"""ak.wwise.core.transport"""
-	
-	def __init__(self, client: _WaapiClient):
-		"""
-		Constructor.
-		:param client: The WAAPI client to use.
-		"""
-		self._client = client
-		
-		# TODO: implement topics
-		self.state_changed: _RefEvent
-	
-	def create(self):
-		"""
-		Creates a transport object for the given Wwise object. The return transport object can be used to
-		play, stop, pause and resume the Wwise object via the other transport functions.
-		"""
-	
-	def destroy(self):
-		"""
-		Destroys the given transport object.
-		"""
-	
-	def execute_action(self):
-		"""
-		Executes an action on the given transport object, or all transport objects if none is specified.
-		"""
-	
-	def get_list(self):
-		"""
-		Returns the list of transport objects.
-		"""
-	
-	def get_state(self):
-		"""
-		Gets the state of the given transport object.
-		"""
-	
-	def prepare(self):
-		"""
-		Prepare the object and its dependencies for playback. Use this function before calling
-		`PostEventSync` or `PostMIDIOnEventSync` from `IAkGlobalPluginContext`.
-		"""
-	
-	def use_originals(self):
-		"""
-		Sets the Original/Converted transport toggle globally. This allows playing the original or the
-		converted sound files.
-		"""
+    """ak.wwise.core.transport"""
+    
+    def __init__(self, client: _WaapiClient):
+        """
+        Constructor.
+        :param client: The WAAPI client to use.
+        """
+        self._client = client
+        
+        # TODO: implement topics
+        self.state_changed: _RefEvent
+    
+    def create(self, wwise_object: tuple[EObjectType, Name] | GUID | ProjectPath,
+               game_object: GameObjectID = None) -> int:
+        """
+        https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_transport_create.html \n
+        Creates a transport object for the given Wwise object. The return transport object can be used to
+        play, stop, pause and resume the Wwise object via the other transport functions.
+        :arg wwise_object: The ID (GUID), name, or project path of the object to control via the transport object.
+        :arg game_object: The game object to use for playback.
+        :return: Transport object ID to be used with all other ak.wwise.core.transport functions. Returns -1 if function
+        call failed.
+        """
+        if wwise_object is None:
+            return -1
+        
+        args = {"object": wwise_object if not
+                isinstance(wwise_object, tuple) else f"{wwise_object[0]}_{wwise_object[1]}"}
```

Comment:
This won't work! An `EObjectType` value is actually a `tuple` of 3 elements. In this case, you need to replace `wwise_object[0]` with `wwise_object[0].get_type_name()`.